README.md

```diff
@@ -46,129 +45,129 @@ To run locally:
-    ModelBenchmark.firstHashCode    groovystatic  avgt   30   726.060 ±  5.983  ns/op
-    ModelBenchmark.firstHashCode          lombok  avgt   30    45.172 ±  0.581  ns/op
-    ModelBenchmark.firstHashCode           scala  avgt   30    68.644 ±  0.498  ns/op
-    ModelBenchmark.firstHashCode          kotlin  avgt   30    41.501 ±  0.427  ns/op
-    ModelBenchmark.firstHashCode      kotlinlazy  avgt   30   125.924 ±  1.734  ns/op
-    ModelBenchmark.firstHashCode       autovalue  avgt   30    42.409 ±  0.580  ns/op
-    ModelBenchmark.firstHashCode           guava  avgt   30   108.756 ±  1.352  ns/op
+    Benchmark                            (model)  Mode  Cnt     Score     Error  Units
+    ModelBenchmark.firstHashCode        intellij  avgt   30    83.722 ±  20.511  ns/op
+    ModelBenchmark.firstHashCode         eclipse  avgt   30    45.636 ±   0.279  ns/op
+    ModelBenchmark.firstHashCode          groovy  avgt   30   174.447 ±   0.580  ns/op
+    ModelBenchmark.firstHashCode    groovystatic  avgt   30  3887.561 ± 219.875  ns/op
+    ModelBenchmark.firstHashCode          lombok  avgt   30    44.019 ±   1.494  ns/op
+    ModelBenchmark.firstHashCode           scala  avgt   30   147.650 ±  29.696  ns/op
+    ModelBenchmark.firstHashCode          kotlin  avgt   30    55.645 ±  19.007  ns/op
+    ModelBenchmark.firstHashCode      kotlinlazy  avgt   30   230.978 ±  27.861  ns/op
+    ModelBenchmark.firstHashCode       autovalue  avgt   30    46.362 ±   2.225  ns/op
+    ModelBenchmark.firstHashCode           guava  avgt   30    84.607 ±   1.704  ns/op
 
 ## hashCode()
 
-    Benchmark                            (model)  Mode  Cnt     Score    Error  Units
-    ModelBenchmark.hashCode             intellij  avgt   30    25.924 ±  0.200  ns/op
-    ModelBenchmark.hashCode              eclipse  avgt   30    27.128 ±  0.225  ns/op
-    ModelBenchmark.hashCode               groovy  avgt   30   790.066 ±  7.902  ns/op
-    ModelBenchmark.hashCode         groovystatic  avgt   30   705.565 ±  7.212  ns/op
-    ModelBenchmark.hashCode               lombok  avgt   30    27.113 ±  0.300  ns/op
-    ModelBenchmark.hashCode                scala  avgt   30    45.070 ±  0.705  ns/op
-    ModelBenchmark.hashCode               kotlin  avgt   30    27.024 ±  0.255  ns/op
-    ModelBenchmark.hashCode           kotlinlazy  avgt   30     4.101 ±  0.039  ns/op
-    ModelBenchmark.hashCode            autovalue  avgt   30    25.372 ±  0.210  ns/op
-    ModelBenchmark.hashCode                guava  avgt   30    92.790 ±  1.038  ns/op
+    Benchmark                            (model)  Mode  Cnt     Score     Error  Units
+    ModelBenchmark.hashCode             intellij  avgt   30    23.425 ±   0.195  ns/op
+    ModelBenchmark.hashCode              eclipse  avgt   30    23.948 ±   0.208  ns/op
+    ModelBenchmark.hashCode               groovy  avgt   30   130.400 ±   3.738  ns/op
+    ModelBenchmark.hashCode         groovystatic  avgt   30  3175.860 ±  58.442  ns/op
```

Comment:
Uh oh. Not that it was great when compared to the other implementations, but still much worse.